projectgenerator/gui/ili2db_options.py

```diff
@@ -33,24 +38,46 @@ def __init__(self, parent=None):
 
         self.buttonBox.accepted.disconnect()
         self.buttonBox.accepted.connect(self.accepted)
+        self.buttonBox.rejected.disconnect()
+        self.buttonBox.rejected.connect(self.rejected)
+        self.toml_file_browse_button.clicked.connect(
+            make_file_selector(self.toml_file_line_edit, title=self.tr('Open TOML File'),
+                               file_filter=self.tr('Toms Obvious Minimal Language File (*.toml)')))
```

Comment:
I think a user is more interested in the content of the file. Something like "Interlis Extra Meta Attributes" (other proposals also welcome).

Reply:
I agree. But isn't it clear to everyone that it's an *Interlis* Extra Meta Attributes? 
What do you think about "Extra Attributes File (*.toml)"?